src/components/10-atoms/checkbox/icon/index.js

```diff
@@ -0,0 +1 @@
+export { default as TickHardEdgesSvg } from './tick-hard-edges.svg';
```

Comment:
Instead of creating all those files, I would rather inline the svg code into the template (or create a svg variable in the checkbox index.js file).

Reply:
why? better that it can be reused somewhere else IMHO

Reply:
I reduced it to one file for the esport, Because I really don't like the svg's in the code. They don't wrap and this looks kinda ugly in my opinion.

Reply:
@LucaMele Noah said this icon won't be used anywhere else and is stored within the component.

But I have no strong opinion here. :)